assets/dev/js/editor/atomic-widgets/models/a-container.js

```diff
@@ -0,0 +1,10 @@
+import { default as ElementModel } from '../../elements/models/element';
+
+export default class Container extends ElementModel {
+	/**
+	 * Do not allow section or column be placed in the container.
+	 */
+	isValidChild() {
+		return true;
```

Comment:
You should have actual logic here instead of just `true`?

Reply:
Actually the logic is it. As by default the widget can not contain any element and we will return false there in case of widgets, but as in our case we are using non widget type, and it is a div like a container, so that we need just return true there to have an ability to add child elements there

Reply:
@baghdasarovelementor I was expecting something like this:

```
/**
 * Do not allow section, column or container be placed in the Atomic container.
 *
 * @param {*} childModel
 */
isValidChild( childModel ) {
	const elType = childModel.get( 'elType' );

	return 'section' !== elType && 'column' !== elType && 'container' !== elType;
}
```